### DISCUSSION TITLE
Allow reminders to be set on a specific date and/or remove the artificial 365 day limit

### DISCUSSION THREAD

**Discussion by drollinger** [Infisical/infisical#6310] [Feature Request]
### Feature description
A clear and concise description of what the feature should be.

I have a credential that expires every 2 years. So the artificial limit of 365 days is a bit annoying. It would also be nice to be able to specify the day I need the reminder to go off since if I create the reminder a week after I built the credential and set it for lets say 2 years, it's not going to go off until after it already expired. So it would be nice to specify and also see when the reminder will go off next.

### Why would it be useful?
Why would this feature be useful for Infisical users? 

Credentials have expiration times that exceed 1 year. Also, it's common to set up a reminder on a different day then the day you created your credential.

**Comment by akhilmhdh** [comment]
Moving this to [feature request discussion](https://github.com/Infisical/infisical/issues/5172).
